src/cookies.md.njk

```diff
@@ -1,7 +1,118 @@
+  head: [
+    { text: "Name" },
+    { text: "Purpose"},
+    { text: "Expires"}
+  ],
+  rows: [
+    [
+      {text: "design-system-cookie-policy"},
+      {text: "Saves your cookie consent settings"},
+      {text: "1 year"}
+    ]
+  ]
+}) }}
+
+
+## Analytics cookies (optional)
+
+We use Google Analytics software to understand how people use the GOV.UK Design
+System. We do this to help make sure the site is meeting the needs of its users
+and to help us make improvements.
+
+We do not collect or store your personal information (for example your name or
+address) so this information cannot be used to identify who you are.
+
+We do not allow Google to use or share our analytics data.
+
+Google Analytics stores information about:
+
+  - the pages you visit
+  - how long you spend on each page
+  - how you arrived at the site
+  - what you click on while you have visiting the site
+  - the device and browser you are using
+
+Google Analytics sets the following cookies:
+
+{{ govukTable({
+  firstCellIsHeader: true,
+  head: [
+    { text: "Cookie name" },
+    { text: "Default expiration time" },
+    { text: "Description" }
+  ],
+  rows: [
+    [
+      { text: "_ga" },
+      { text: "2 years" },
+      { text: "Used to distinguish users." }
+    ],
+    [
+      { text: "_gid" },
+      { text: "24 hours" },
+      { text: "Used to distinguish users." }
+    ],
+    [
+      { text: "_gat" },
+      { text: "1 minute" },
+      { text: "Used to throttle request rate. If Google Analytics is deployed via Google Tag Manager, this cookie will be named _dc_gtm_<property- id>." }
+    ],
+    [
+      { text: "__utma" },
+      { text: "2 years from set/update" },
+      { text: "Used to distinguish users and sessions. The cookie is created when the javascript library executes and no existing __utma cookies exists. The cookie is updated every time data is sent to Google Analytics." }
+    ],
+    [
+      { text: "__utmt" },
+      { text: "10 minutes" },
+      { text: "Used to throttle request rate." }
+    ],
+    [
+      { text: "__utmb" },
+      { text: "30 mins from set/update" },
+      { text: "Used to determine new sessions/visits. The cookie is created when the javascript library executes and no existing __utmb cookies exists. The cookie is updated every time data is sent to Google Analytics." }
+    ],
+    [
+      { text: "__utmc" },
+      { text: "End of browser session" },
+      { text: "Not used in ga.js. Set for interoperability with urchin.js. Historically, this cookie operated in conjunction with the __utmb cookie to determine whether the user was in a new session/visit." }
+    ],
+    [
+      { text: "__utmz" },
+      { text: "6 months from set/update" },
+      { text: "Stores the traffic source or campaign that explains how the user reached your site. The cookie is created when the javascript library executes and is updated every time data is sent to Google Analytics." }
+    ],
+    [
+      { text: "__utmv" },
```

Comment:
Not sure where these cookie names have come from? Looking at the Google Developer docs, I think these apply to an old/different way of implementing Google Analytics which doesn't apply to us.

https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage

Reply:
Not sure, I just took them from the doc https://docs.google.com/document/d/1_Qxe-EMACEb-RDaX3Z0Kyu9tA1caiQc7B5n8ZzcCG5M/edit#

Reply:
We're unsure here, Sara suggested looking at PaaS and their GA implementation?
It's possible it's leftover from creating the template policy from PaaS.